src/modules/render-list/listeners.js

```diff
@@ -70,15 +70,31 @@ export default class Listeners extends DisplayList {
       this.todoList.tasks[id].description = input.value;
       UpdateList.saveList(this.todoList);
     });
-  }
+  };
+
+  checkTask = (task) => {
+    const li = task.parentNode;
+    const checkBtn = li.querySelector('.check');
+    const id = Number(task.id.replace(/\D+/g, ''));
+
+    if (!li.classList.contains('task-completed')) {
+      checkBtn.addEventListener('click', () => {
+        if (id) {
+          this.todoList = UpdateList.completeTask(id, this.todoList);
+          this.showAll();
+        }
+      });
+    }
```

Comment:
- [ ] Your app correctly updates the `completed` value to `true` in the localStorage and adds a checkmark for completed tasks, however when the checkbox is **clicked again** to uncheck the box and update the `completed` property to `false`, this does not update in the localStorage and the same styling remains. The `completed` property should be correctly updated to `true` or `false` upon clicking the checkbox. Kindly fix this.

  ![leo-completed](https://user-images.githubusercontent.com/36542986/154541217-62551060-0d30-4b1d-a55c-8421eae88fff.jpg)